### PR TITLE
fix: Conferencing Label When Installing Apps

### PR DESCRIPTION
**Pull request by joeauyeung**
## What does this PR do?

Disables install conferencing apps on teams & orgs until the PR is ready.

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- Create a team or org
- Install a video app
    - The only option should be to install the app to the user

## Mandatory Tasks

- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: apps/web/components/apps/App.tsx [hunk 1]
```diff
@@ -416,7 +416,7 @@ const InstallAppButtonChild = ({
     },
   });
 
-  if (!userAdminTeams?.length || appCategories.some((category) => category === "calendar")) {
+  if (!userAdminTeams?.length || appCategories.some((category) => category === "calendar" || category === "conferencing")) {
```
Comment (zomars) [inline apps/web/components/apps/App.tsx:419]:
```suggestion
  if (!userAdminTeams?.length || appCategories.some((category) => ["calendar", "conferencing"].includes(category))) {
```

**Review by zomars — DISMISSED** [review]
Can we check why there are type errors in the PR? 🙏🏽

File: packages/ui/components/apps/AppCard.tsx [hunk 2]
```diff
@@ -37,9 +37,11 @@ export function AppCard({ app, credentials, searchText, userAdminTeams }: AppCar
 
   const allowedMultipleInstalls = app.categories && app.categories.indexOf("calendar") > -1;
   const appAdded = (credentials && credentials.length) || 0;
-  const appInstalled = userAdminTeams?.length
-    ? userAdminTeams.length && appAdded >= userAdminTeams.length
-    : appAdded > 0;
+  const enabledOnTeams = !app.categories.some(
+    (category) => category === "calendar" || category === "conferencing"
+  );
```
Comment (joeauyeung) [inline packages/ui/components/apps/AppCard.tsx:42]: @zomars, I need to add a check for concurrent meetings when enabling team conferencing apps. This is just a part of the stack of PRs that would be nice to have in prod.

File: packages/ui/components/apps/AppCard.tsx [hunk 3]
```diff
@@ -37,9 +37,11 @@ export function AppCard({ app, credentials, searchText, userAdminTeams }: AppCar
...
+
+  const appInstalled = enabledOnTeams && userAdminTeams ? userAdminTeams.length === appAdded : appAdded > 0;
```
Comment (hariombalhara) [inline packages/ui/components/apps/AppCard.tsx:44]: nit: `isAppInstalledForAllTargets = userAdminTeams.length === appAdded`

**Review by hariombalhara — APPROVED** [review]

**Pull request merged**